features/fixtures/mazerunner/app/build.gradle

```
@@ -8,13 +8,14 @@ android {
     ndkVersion parent.ext.ndkVersion
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 17
         targetSdkVersion 31
         versionCode 34
         versionName "1.1.14"
         manifestPlaceholders = [
                 // omit any of the following placeholders to use the default values
                 bugsnagApiKey: "abc12312312312312312312312312312",
+                bugsnagPerformanceApiKey: "abc12312312312312312312312312312",
```

we should allow this to be set as an environment variable:

```suggestion
                bugsnagPerformanceApiKey: System.env.BUGSNAG_PERFORMANCE_API_KEY ?: "abc12312312312312312312312312312",
```